helpers/convertDis.js

```diff
@@ -0,0 +1,34 @@
+const convertoption = (item) => {
+  item = item.toLowerCase()
+  var options = []
+  if (item === 'visual') {
+    options = ['Audio Recordings', 'Braille', 'Big Fonts', 'Carer', 'Place for Guide Dog', 'SMS messaging']
+  }
+  if (item === 'aural') {
```

Comment:
Is there a reason that you use if and not if/else statements, and then return in each section?

You could do:
```
  if (item === 'mental') {
    return  ['Good Lighting', 'Carer']
  }
  else (item === 'physical') {
    return ['WheelChair Access', 'Disabled Parking', 'Disabled Toilets', 'Carer']
  }
```